corehq/apps/hqwebapp/static/hqwebapp/js/hq.helpers.js

```diff
@@ -155,6 +155,7 @@ hqDefine("hqwebapp/js/hq.helpers", [
         }
         ko.applyBindings(context, this.get(0));
         this.removeClass('ko-template');
+        $('.ko-loading').addClass('hide');
```

Comment:
This doesn't belong here until `.ko-loading` is a pattern instead of a one-off.

Reply:
can it be a pattern? :-)

Reply:
sure, just go apply it in a half-dozen other pages and boom it's a pattern ;)